### DISCUSSION TITLE
Settings and configs for VSCode

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/spmssite#13] [Q&A] [answered]
I know its not typically reccomended to add settings, workspaces and configurations to source control, but I tried to provide an intial setup that would help someone get working on this project easily if they had no working configuration or settings to go on which would ultimately lead to the best product.

These would only be aimed towards newer developers.

I feel it would be helpful.  

Ideas, comments questions, concerns, thoughts?

**Comment by pranavmadhu01** [comment, marked as answer]
I prefer vs code for editing 
To get started with development in vs code especially for Web-Dev I recommend newbies to install the following extensions:-
Prettier
Live server
auto complete tag
auto rename tag
path intellisense
git lens

**Reply by gbowne1** [reply]
Yes I work in VSCode although I have just about ever other common editor/IDE installed.

**Comment by gbowne1** [comment]
I added the appropriate settings via merge today.

Add any you feel are appropriate and PR.
